packages/dd-trace/test/appsec/iast/analyzers/unvalidated-redirect-analyzer.express.plugin.spec.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const { UNVALIDATED_REDIRECT } = require('../../../../src/appsec/iast/vulnerabilities')
+const { prepareTestServerForIastInExpress } = require('../utils')
+const { storage } = require('../../../../../datadog-core')
+const iastContextFunctions = require('../../../../src/appsec/iast/iast-context')
+const { newTaintedString } = require('../../../../src/appsec/iast/taint-tracking/operations')
+
+describe('Unvalidated Redirect vulnerability', () => {
+  let redirectFunctions
+  const redirectFunctionsFilename = 'redirect-express-functions.js'
+  const redirectFunctionsPath = path.join(os.tmpdir(), redirectFunctionsFilename)
+
+  before(() => {
+    fs.copyFileSync(path.join(__dirname, 'resources', redirectFunctionsFilename), redirectFunctionsPath)
+    redirectFunctions = require(redirectFunctionsPath)
+  })
+
+  after(() => {
+    fs.rmSync(redirectFunctionsPath)
```

Comment:
use `unlinkSync` instead of `rmSync`. This one is not available in old node versions.